services/QuillLMS/engines/evidence/lib/evidence/evidence/check.rb

```diff
@@ -20,44 +22,61 @@ module Check
       optimal: true,
     }
 
-    def self.get_feedback(entry, prompt, previous_feedback)
-      triggered_check = find_triggered_check(entry, prompt, previous_feedback)
+    def self.get_feedback(entry, prompt, previous_feedback, feedback_types=nil)
+      triggered_check = find_triggered_check(entry, prompt, previous_feedback, feedback_types)
 
-      triggered_check&.response || fallback_feedback
+      triggered_check || fallback_feedback
     end
 
     # returns first nonoptimal feedback, and if all are optimal, returns automl feedback
```

Comment:
Please mention in the comment that this function can also return `nil`, as that's an important code path.